### PR TITLE
STRATCONN-3173: Add dropdown to region setting

### PR DESCRIPTION
**Pull request by maryamsharif**
Making region setting in The Trade Desk Destination a dropdown so it is easier for customers to configure.

## Testing
Screenshot 2024-05-20 at 11 27 45 AM

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [x] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/destination-actions/src/destinations/the-trade-desk-crm/index.ts [hunk 1]
```diff
@@ -77,7 +77,12 @@ const destination: AudienceDestinationDefinition<Settings, AudienceSettings> = {
       label: 'Region',
       description:
         'The geographical region of the CRM data segment based on the origin of PII. Can be US (United States and Canada), EU (European Union and the UK), or APAC (Asia-Pacific)',
-      required: true
+      required: true,
+      choices: [
```
Comment (marinhero) [inline packages/destination-actions/src/destinations/the-trade-desk-crm/index.ts:81]: Hi! Mind sharing how did you verify these are valid values?

Comment (maryamsharif) [reply packages/destination-actions/src/destinations/the-trade-desk-crm/index.ts:81]: The [documentation](https://partner.thetradedesk.com/v3/portal/api/ref/post-crmdata-segment) for TTD states that these are the only 3 values. Screenshot 2024-05-21 at 10 14 19 AM

**Comment by brennan** [comment]
@maryamsharif The code looks good to me! Because `region` was already a required setting, we also won't have to run a migration. Not a blocker—but what happens if a customer previously entered a random string (i.e. not one of the three accepted region strings) into the setting?

**Review by brennan — APPROVED** [review]

**Comment by joe-ayoub-segment** [comment]
Hi @maryamsharif - deploy complete - no push done. Please confirm the change looks good.

**Pull request merged**